ws4redis/context_processors.py

```diff
@@ -7,10 +7,11 @@ def default(request):
     """
     Adds additional context variables to the default context.
     """
+    host = settings.WEBSOCKET_HOST or request.get_host()
```

Comment:
Should we use `getattr(settings, 'WEBSOCKET_HOST', None) or request.get_host()` here too in case it isn't defined?